README.md

```diff
@@ -76,22 +78,24 @@ Click on `Add notebooks` (upper right) to create a notebook.
 ## 4. Add the data and configuration file
 
 #### Add the data and configuration to the notebook
-Use `Find and Add Data` (look for the `10/01` icon)
-and its `Files` tab. From there you can click
-`browse` and add data and configuration files from your computer.
 
-> Note:  If you don't have your own data and configuration files, you can get our example by cloning
-this git repo. Look in the `data/sample_text.txt` directory for data file and in the `configuration/sample_config.txt`.
-If you use configuration file from your computer, make sure to conform to the JSON structure given in `configuration/sample_config.txt`.
+* From the `My Projects > Default` page, Use `Find and Add Data` (look for the `10/01` icon)
+and its `Files` tab. 
+* Click `browse` and navigate to this repo `watson-document-classifier/data/sample_text.txt
```

Comment:
Looks like this line and next line are missing the ending back-tick around the file names